src/search/search-feedback.js

```diff
@@ -0,0 +1,81 @@
+import { getUser, setUser } from "./user.js";
+import { trackEvent } from './tracking.js';
+
+export function getSearchFeedbackEmbed(numHits) {
+  if (numHits === 0) {
+    // Set this key so we don't show them the popup later
+    window.sessionStorage.setItem(SEARCH_FEEDBACK_SHOW_STORAGE_KEY, "yes");
+
```

Comment:
For GA tracking would enter:
`trackEvent("feedback-modal", "triggered", "zero-results");`